packages/backend/src/api/routers/TvlRouter.ts

```diff
@@ -57,14 +57,22 @@ export function createTvlRouter(
 
   if (features.detailedTvlEnabled) {
     router.get('/api/detailed-tvl', async (ctx) => {
-      const data = await detailedTvlController.getDetailedTvlApiResponse()
+      const detailedTvlResult =
+        await detailedTvlController.getDetailedTvlApiResponse()
+
+      if (detailedTvlResult.result === 'error') {
```

Comment:
I think I'd prefer a more holistic approach to error handling and codes in the controller. This can happen as another PR

Reply:
We need to standardize it either way. Can't estimate how many endpoints' backend will grow to. Now we have 'some' convention which relies on partly handled cases and exceptions I suppose. Another PR it is 😅